backend/src/models/game.model.ts

```diff
@@ -15,20 +15,22 @@ export enum GameState {
 
 export interface Game extends Document {
   gameCode: string;
-  settings?: Settings;
+  settings: Settings;
   setups: Types.DocumentArray<Setup>;
   discardedSetups?: Types.DocumentArray<Setup>;
   punchlines: string[];
   discardedPunchlines?: string[];
   players: Types.DocumentArray<Player>;
-  host?: ObjectId;
   state?: GameState;
   rounds?: Types.DocumentArray<Round>;
 }
 
 const GameSchema: Schema = new Schema({
   gameCode: { type: String, required: true, unique: true },
-  settings: SettingsSchema,
+  settings: {
+    type: SettingsSchema,
+    default: () => ({}),
+  },
```

Comment:
What is this change doing? (ie. is this necessary for this PR?)

Reply:
This was necessary as the settings object was not set by default on a new Game